packages/block-library/src/accordions/style.scss

```diff
@@ -0,0 +1,92 @@
+.wp-block-accordion-item {
+	display: grid;
+	grid-template-rows: max-content 0fr;
+}
+
+.wp-block-accordion-item.is-open {
+	grid-template-rows: max-content 1fr;
+}
+
+.wp-block-accordion-item .accordion-item__heading {
```

Comment:
It looks like these lines are to prevent heading element styles from bleeding into this block. The problem is that they prevent the global styles for the block from working. 

FWIW I think it's ok for this block to inherit heading element styles, since it uses an element. If designers want to only target heading blocks they can do so.

Reply:
I think it'd best to keep the margin controls on these blocks, so I've tried making the CSS less specific in https://github.com/WordPress/gutenberg/pull/64119/commits/43dc276227066b8e9b59be99873d9dedca8f9b5d. This means any margins set in Global Styles should be applied correctly.

Reply:
It works!